<launch>
  <node pkg="swri_transform_util" exec="initialize_origin.py" name="initialize_origin" >
    <param name="local_xy_frame" value="map"/>
    <param name="local_xy_origin" value="swri"/>
    <param name="local_xy_origins" value="
      [{ name: swri,
         latitude: 29.45196669,
         longitude: -98.61370577,
         altitude: 233.719,
         heading: 0.0},
       
       { name: back_40,
         latitude: 29.447507,
         longitude: -98.629367,
         altitude: 200.0,
         heading: 0.0}]
         "/>
  </node>
</launch>

